automatic_cross_compile.sh

```diff
@@ -0,0 +1,61 @@
+# This script is intended to be used by the Jenkins night builds.
+# But it still possible to use manually setting up this variables:
+#
+# TARGET=[raspbian]
+# ROS2_DISTRO=[ardent,bouncy,crystal,master,...]
```

Comment:
Is there a list on ROS2 documentation with all the ROS2_DISTRO available ?
It looks like this value can be any of the branches defined in the https://github.com/ros2/ros2/blob/master/ros2.repos repository.

Reply:
Yes, for the script it's sufficient to provide any of the branches (which are more than the available ros2 distributions).
I don't think there is such a list...